lib/ProductItem.dart

```diff
@@ -0,0 +1,60 @@
+class ProductItem {
+  String? calories;
+  String? carbos;
+  String? description;
+  int? difficulty;
+  String? fats;
+  String? headline;
+  String? id;
+  String? image;
+  String? name;
+  String? proteins;
+  String? thumb;
+  String? time;
+
+  ProductItem(
+      {this.calories,
+        this.carbos,
+        this.description,
+        this.difficulty,
+        this.fats,
+        this.headline,
+        this.id,
+        this.image,
+        this.name,
+        this.proteins,
+        this.thumb,
+        this.time});
```

Comment:
- Always create an object as immutable. See more [why?](https://medium.com/@yetesfadev/immutable-in-dart-and-flutter-understanding-usage-and-best-practices-742be5fa25ea)
- Each type should never be nullable if we know that its values are not null